### PR TITLE
Masoud: Image label verification 

### PR DESCRIPTION
**Pull request by masoudbigdeli**

### CODE REVIEW

File: crowd-ui/src/features/image-label-verification/ImageLabelVerification.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+              </a>
+            </div>
+          </div>
+        </div>
+      )}
+    </>
+  );
+};
+
+export default ImageLabelQuestions;
```
Comment (masoudbigdeli) [inline crowd-ui/src/features/image-label-verification/ImageLabelVerification.tsx:125]: this is the file that causes Error

File: crowd-ui/.prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 100
+}
```
Comment (kimiaShahbaghi) [inline crowd-ui/.prettierrc]: is this file necessary ?

File: crowd-ui/src/features/home/Home-page.tsx [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
-
-
-export default Home;
\ No newline at end of file
+import { Link } from 'react-router-dom';
+
+const Home = () => (
+  <div>
+    <Link to="/imageLabelVerification">
+      <button type="submit">Image Label Verification</button>
```
Comment (kimiaShahbaghi) [inline crowd-ui/src/features/home/Home-page.tsx]: why type submit?

**Review by kimiaShahbaghi — APPROVED** [review]

File: crowd-ui/.prettierrc (hunk 2 above)
Comment (masoudbigdeli) [reply crowd-ui/.prettierrc]: No, it is deleted in last commit.

File: crowd-ui/src/features/home/Home-page.tsx (hunk 3 above)
Comment (masoudbigdeli) [reply crowd-ui/src/features/home/Home-page.tsx]: corrected in last commit.

File: crowd-ui/src/features/image-label-verification/components/Header/Header.tsx [hunk 4]
```diff
@@ -1,19 +1,44 @@
...
+            }}
+          >
+
+            <MoreVertIcon />
+          </IconButton>
+        </Link>
+
+        <h3 className="text-style"><FormattedMessage id="imageLabel.appName" /></h3>
+
+        <Link to="/imageLabelVerification">
```
Comment (kimiaShahbaghi) [inline crowd-ui/src/features/image-label-verification/components/Header/Header.tsx]: shouldn't we use history?

File: crowd-ui/src/helper/requestConfig.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+const request = axios.create({
+  baseURL: 'https://react-http-8ae62-default-rtdb.firebaseio.com',
```
Comment (ketabi) [inline crowd-ui/src/helper/requestConfig.js]: Move this to .env please

**Review by samaneteymurifar — APPROVED** [review]

**Review by SBorouny — APPROVED** [review]
